src/test/parse.ts

```diff
@@ -114,6 +118,25 @@ export function parseErrorsFromMarkup(text: string): LintError[] {
     return lintErrors;
 }
 
+function substituteMessage(substitutions: Map<string, string>, message: string): string {
```

Comment:
can you rename `substitutions` to `templateMap`? It's confusing because the values that are put into the template are also substitutions. Maybe also rename `message` to something like `templateCall`